app/controllers/assignments_controller.rb

```diff
@@ -138,4 +135,28 @@ def start_upload(assignment, file)
       # Launch java program to process submissions
       SubmissionsHandler.process_submissions(submissions_path, assignment)
   end
+
+  def is_valid_zip?(memeType, filePath)
+    print filePath
+    print memeType 
+    if memeType == "application/x-zip-compressed" || 
+      memeType == "application/zip-compressed" ||
+      memeType == "application/zip"
+      return true;
+    else
+      return is_opened_as_zip?(filePath)
+    end
+  end
+
+  def is_opened_as_zip?(path)
```

Comment:
Generally, you need some inlined documentation.  Please add at least function level documentation (input/output/author).

Reply:
You can follow PEP guidelines (from Google)

Reply:
The documentation has been included as requested. I am using the RDoc documentation style as recommended online.

Reply:
Thanks!